.github/workflows/c-cpp.yml

```
@@ -233,30 +182,59 @@ jobs:
+        __cflags: ${{ env.__cflags }} --target=x86_64-windows-msvc
+        __exe: ${{ env.__exe }}.exe
+      working-directory: ${{ github.workspace }}/${{ env.__src_path }}
+      run: |
+        BIN="$__msvc_sdk_dir/bin/x64" . "$__msvc_dir/msvcenv-native.sh"
+        "$__cc" $__cflags -o "$__exe" $__c_src
+
+    - name: Archive build artifacts
       uses: actions/upload-artifact@v4
       with:
-        name: dbs26-macos
+        name: dbs26-${{ matrix.os }}${{ matrix.msvc ? '+windows' : '' }}
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 223: mapping values are not allowed in this context

(syntax-check)